From ce76f7b8411960d897de90d1de44e564d05dc543 Mon Sep 17 00:00:00 2001
From: "Ooi, Joyce" <joyce.ooi@intel.com>
Date: Fri, 29 Nov 2019 23:26:42 +0800
Subject: [PATCH 125/151] MAINTAINERS: Add entry for Intel FPGA Quad Speed
 Ethernet Driver

commit  87147b0384f4cbc7f2dba8d9d583f5635bf931ad from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Add a MAINTAINERS entry covering the Intel FPGA Quad Speed Ethernet Driver

Signed-off-by: Ooi, Joyce <joyce.ooi@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 MAINTAINERS | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index f5e7745c6d4b..fe78de56f9c3 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7942,6 +7942,13 @@ F:	drivers/net/ethernet/intel/
 F:	drivers/net/ethernet/intel/*/
 F:	include/linux/avf/virtchnl.h
 
+INTEL FPGA QUAD SPEED ETHERNET DRIVER
+M:      Joyce Ooi <joyce.ooi@intel.com>
+L:      netdev@vger.kernel.org
+S:      Maintained
+F:      Documentation/devicetree/bindings/net/intel_fpga_qse.txt
+F:      drivers/net/ethernet/altera/intel_fpga_qse_ll
+
 INTEL FRAMEBUFFER DRIVER (excluding 810 and 815)
 M:	Maik Broemme <mbroemme@libmpq.org>
 L:	linux-fbdev@vger.kernel.org
-- 
2.26.1

